### ISSUE TITLE
"Very minor typo list" from reviewer

### ISSUE THREAD

**Issue by taylori** [cgrandin/hake-assessment#281]
List below was provided by Trevor. I believe that some of these are style choices, not errors, such as the y-axis not starting at 0 in some plots. Other clearly should be fixed. I'm happy to take a first crack at this, but if somebody else wants to go first, that's fine too. I'm not sure how best to go about checking these off without opening a whole bunch of issues. Perhaps just open issues for the ones that seem more questionable.

> Hi Ian: 
> 
> Reading through the assessment carefully, I noted all the very minor typos. Obviously none need to be corrected or make a difference to the assessment. On the other hand, maybe changing them in the auto-production of the assessment means they will be corrected in all future years! 
> 
> So do with these whatever you want! 
> 
> Trevor
> 
> ======================================================
> 
> p. 22 “first formed in 2011 and 2012 was the first year” should be “first formed in 2011, and 2012 was the first year”
> p. 22 “Pacific Hake (or Pacific whiting” here whiting should be capitalized, presumably.
> p. 24 “Pacific Hake , also referred to as Pacific whiting ,” remove the spaces before each of the commas.
> p. 24 “family Merluccidae”, the family name should not be in italics.
> p. 25 “((Ressler et al., 2007; Hamel et al., 2015)).” Remove the extra ().
> p. 25 “as estiamted in this assessment” should be “estimated”
> p. 28 “was further divided to” should perhaps be “was further divided among the”
> p. 28 “on September 15 ,” delete space before comma.
> p. 28 and elsewhere, no space before “N” in latitude, and no period after “N”.
> p. 28 “regulations don’t” should be “regulations do not”
> p. 28 “was substantially greater then in” should be “was substantially greater than in”
> p. 29 “weight-at-age from fishery and survey catches,” delete the ending comma.
> p. 31 “nearly equal in 2016 .” delete the space before the period.
> p. 35 “782 individual ovary collections” does this mean 782 individual ovaries, or 782 sets of ovaries collected at different times?
> p. 35 “relative contribute to the female spawning biomass” should be “contribution” not “contribute”
> p. 35 “At-Sea hake Observer” here “Hake” should be capitalized.
> p. 36 “south of 34.5 degrees latitude” in two places, should be “south of 34.5°S”
> p. 41 “have surely been subject to” delete “surely”.
> p. 41 The sentence starting “Selectivity has also been modeled in several ways: it has been invariant...” does not make grammatical sense.
> p. 43 “the the processing” remove the duplicated “the”.
> p. 46 “and to‘tune’ the 2017 base model” add a space after “to”.
> p. 46 “little autocorrelation (Figures 16 and 17.” needs to end with “17).”.
> p. 49 “comprising of 47%” should be “comprising 47%”
> p. 49 “suggests that is one of” should be “suggests that it is one of”
> p. 49 “High and low recruitment has been produced” the “has” should be “have”.
> p. 49 “2-year-old fish in the 2012” should be “2-year-old fish in 2012”
> p. 49 “has been below” should be “was below”
> p. 50 “in all years except and 1999” a year is missing after “except”.
> p. 52 “relative to 100% target” should be “relative to the 100% target”
> p. 52 “The probability that the 2018 spawning biomass will be less than the 2017 spawning biomass is 82% or less for all catch levels” Perhaps better to say: “... is high, up to 82% depending on catch levels”.
> p. 53 “2017 estimates at 74.2%” should perhaps be “2017 estimates are 74.2%”
> p. 55 “allows the recruitments to deviated far” should be “deviate” not “deviated”.
> p. 57 “The variability between models” should be “The variability among models” since there are more than two models being compared.
> p. 57 “making sure that” should be “make sure that”
> p. 62 “Hakes of the world (family Merlucciidae).” Here the Merlucciidae should not be italicized since it is a family name and not a genus or species.
> p. 62 “an example of Atlantic cod (Gadus morhua)” species name should be in italics “Gadus morhua”
> Table 5 & 14: numbers should be right aligned.
> Table 15: “Log Rec. deviations” should be “Log rec. deviations”
> Table 17: some columns have a space after the dash.
> Table 20: final column, 27% and 24% should have a number after the decimal place.
> Table 22: “2017 – 2019” no spaces on either side of the dash.
> Figure 38: the upper bound of x-axis needs to be extended so that the 97.5% upper limit is shown.
> Figure 39, 40, 41, 47 and most of the later plots: remove the space between 0 and the axis on the y-axis.

**Comment by aaronmberger** [comment]
Received paper doc from Michelle on minor edits/suggestions, I will incorporate these

**Comment by andrew-edwards** [comment]
I don't mind doing Trevor's ones. I'm not sure exactly what's needed for some of the new tables, so I'm okay spending the time on these edits, as they seem worth attempting. I can just copy this list into a comment here and note which are done or ignored (some are indeed stylistic). 

I'm away on Monday and so can get to them Tuesday. Great job this week - it was great working with you all again.

**Comment by andrew-edwards** [comment]
I'm actually working this afternoon (Monday), probably off Tuesday. I'll go ahead and plough through these. I'll add a comment explaining which I've done or decided not to do.

**Comment by cgrandin** [comment]
Thanks Andy, I'll be back Tuesday morning to work on the document.

**Comment by andrew-edwards** [comment]
Ended up working this afternoon, and will finish these up tomorrow afternoon. Was keeping track in a file trevors-typos.txt, which I've now committed if you want to take a look. Haven't finished yet, will do tomorrow. I've explained why I haven't done something - see the file. (We can delete the file once we're done, and just past the responses into these comments).

**Comment by taylori** [comment]
I made some progress on the list. I think there are only 4 items left, all related to tables:

> Table 15: ¡§Log Rec. deviations¡¨ should be ¡§Log rec. deviations¡¨
> Table 17: some columns have a space after the dash.
> Table 20: final column, 27% and 24% should have a number after the decimal place.
> Table 22: ¡§2017 ¡V 2019¡¨ no spaces on either side of the dash.

I'm getting dragged into some meetings today so won't be as available as I'd wished, but will do my best to keep up and contribute to whatever remaining things need to get done.

**Comment by taylori** [comment]
Also note that the changes to a few figures (extend x-axis limit in one case and remove the space between 0 and the axis on the y-axis in a few others), requires cleaning out the knitr-cache. I tested the individual functions but didn't yet rebuild the whole doc, so hopefully I didn't break anything.

**Comment by andrew-edwards** [comment]
Hmm... the trevor-typos.dat file is a little corrupted (to do with encoding of " I think). Can still use it though. 
I'll take a look at doing those four Table ones that Ian highlighed two comments above.

**Comment by andrew-edwards** [comment]
I've done those four table ones.
I'll clean out my knitr-cache and re-run, to confirm the figure ones are okay (as per Ian's last comment).

**Comment by andrew-edwards** [comment]
Okay, figures build fine and look good, thanks Ian. I'll copy the trevor-typos.dat file here so we can delete it (I'm not going to fix the formatting issues with this text though). We did most of them, and explained why we didn't do certain ones, for the record.

trevor-typos.dat

Minor typos from Trevor.
D = Done.
N = Not done [with explanation why]

[The "s got corrupted somewhere along the way, but this is still readable.]

D - p. 22 [24] ¡§first formed in 2011 and 2012 was the first year¡¨ should be ¡§first formed in 2011, and 2012 was the first year¡¨
N [we discussed last year and I think concluded that Pacific whiting is not the common name, but a colloquial name; also re-ordered for clarity] - p. 22 ¡§Pacific Hake (or Pacific whiting¡¨ here whiting should be capitalized, presumably.
D [actually deleted to avoid repitition] - p. 24 ¡§Pacific Hake , also referred to as Pacific whiting ,¡¨ remove the spaces before each of the commas.
D - p. 24 ¡§family Merluccidae¡¨, the family name should not be in italics.
D - p. 25 ¡§((Ressler et al., 2007; Hamel et al., 2015)).¡¨ Remove the extra ().
D - p. 25 [26] ¡§as estiamted in this assessment¡¨ should be ¡§estimated¡¨
D - p. 28 ¡§was further divided to¡¨ should perhaps be ¡§was further divided among the¡¨
D [comma removed] - p. 28 ¡§on September 15 ,¡¨ delete space before comma.
D - p. 28 and elsewhere, no space before ¡§N¡¨ in latitude, and no period after ¡§N¡¨.
D - p. 28 ¡§regulations don¡¦t¡¨ should be ¡§regulations do not¡¨
D - p. 28 ¡§was substantially greater then in¡¨ should be ¡§was substantially greater than in¡¨
D - p. 29 ¡§weight-at-age from fishery and survey catches,¡¨ delete the ending comma.
D - p. 31 ¡§nearly equal in 2016 .¡¨ delete the space before the period.
D [rephrase for clarity] - p. 35 ¡§782 individual ovary collections¡¨ does this mean 782 individual ovaries, or 782 sets of ovaries collected at different times?
D - p. 35 ¡§relative contribute to the female spawning biomass¡¨ should be ¡§contribution¡¨ not ¡§contribute¡¨
D - p. 35 ¡§At-Sea hake Observer¡¨ here ¡§Hake¡¨ should be capitalized.
D [actually 34.5deg N] -  p. 36 ¡§south of 34.5 degrees latitude¡¨ in two places, should be ¡§south of 34.5¢XS¡¨
N [I think we couldn't be sure, so said 'surely' as a qualifier] - p. 41 ¡§have surely been subject to¡¨ delete ¡§surely¡¨.
N [Ian: I think Trevor was just reading it too fast. Andy: admittedly not the most elegant sentence, but I'm not quite sure the issue - maybe 'invariant' should be 'time-invariant'?] p. 41 The sentence starting ¡§Selectivity has also been modeled in several ways: it has been invariant...¡¨ does not make grammatical sense.
D [and similarly in Exec Summary] p. 43 ¡§the the processing¡¨ remove the duplicated ¡§the¡¨.
D - p. 46 ¡§and to¡¥tune¡¦ the 2017 base model¡¨ add a space after ¡§to¡¨.
D - p. 46 ¡§little autocorrelation (Figures 16 and 17.¡¨ needs to end with ¡§17).¡¨.
D - p. 49 ¡§comprising of 47%¡¨ should be ¡§comprising 47%¡¨
D - [and added , for clarity] p. 49 ¡§suggests that is one of¡¨ should be ¡§suggests that it is one of¡¨
D -  [and 'recruitments' since used plural at start of paragraph] p. 49 ¡§High and low recruitment has been produced¡¨ the ¡§has¡¨ should be ¡§have¡¨.
D - p. 49 ¡§2-year-old fish in the 2012¡¨ should be ¡§2-year-old fish in 2012¡¨
D - p. 49 ¡§has been below¡¨ should be ¡§was below¡¨
D [actually only 1999] - p. 50 ¡§in all years except and 1999¡¨ a year is missing after ¡§except¡¨.
D - p. 52 ¡§relative to 100% target¡¨ should be ¡§relative to the 100% target¡¨
D [the suggestion isn't accurate since our original is a bit unclear/misleading, so just going with "ranges from 17\% to 82\% depending on the catch level" - I think the original phrasing made more sense in previous documents when results were different and could more easily be summarised] p. 52 ¡§The probability that the 2018 spawning biomass will be less than the 2017 spawning biomass is 82% or less for all catch levels¡¨ Perhaps better to say: ¡§... is high, up to 82% depending on catch levels¡¨.
D - p. 53 ¡§2017 estimates at 74.2%¡¨ should perhaps be ¡§2017 estimates are 74.2%¡¨
D - [was already done] - p. 55 ¡§allows the recruitments to deviated far¡¨ should be ¡§deviate¡¨ not ¡§deviated¡¨.
N [not heard of this before, though my office neighbour has and agrees with the change! Not changing since we've probably used 'between' elsehwere, and dictionary says that 'between' can be two or more] p. 57 ¡§The variability between models¡¨ should be ¡§The variability among models¡¨ since there are more than two models being compared.

D [and automatically corrected in Exec Summary also] - p. 57 ¡§making sure that¡¨ should be ¡§make sure that¡¨
D - p. 62 ¡§Hakes of the world (family Merlucciidae).¡¨ Here the Merlucciidae should not be italicized since it is a family name and not a genus or species.
D - p. 62 ¡§an example of Atlantic cod (Gadus morhua)¡¨ species name should be in italics ¡§Gadus morhua¡¨
N [Hard/impossible to do with the vertical lines, which are needed] Table 5: numbers should be right aligned.
D - Table 14: numbers should be right aligned.
D [and spelt out 'recruitment' as there's space] -Table 15: ¡§Log Rec. deviations¡¨ should be ¡§Log rec. deviations¡¨
N [that was done on purporse to keep numbers lining up and the dashes lined up, although it didn't work perfectly (though no spaces would look worse)] -  Table 17: some columns have a space after the dash.
D [needed an f() not a round() - edited other rows also so fixed for future; verified that no numbers changed. Also changes it in Exec Summary table] - Table 20: final column, 27% and 24% should have a number after the decimal place.
D [just the caption; is fine in Exec Summary] - Table 22: ¡§2017 ¡V 2019¡¨ no spaces on either side of the dash.
D - Figure 38: the upper bound of x-axis needs to be extended so that the 97.5% upper limit is shown.
D - [Fixed 40, 41, and 60, skipping plots dependent on r4ss for now] Figure 39, 40, 41, 47 and most of the later plots: remove the space between 0 and the axis on the y-axis.
